site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -24,11 +24,57 @@ permission in the manifest, as shown below:
 
 ## Examples
 
+Opening the extension's onboarding page in new tab after the extension is installed for the first
+time. Must be called from a background context.
+
+```js
+chrome.runtime.onInstalled.addListener((reason) => {
+  if (reason === chrome.runtime.OnInstalledReason.INSTALL) {
+    chrome.tabs.create({
+      url: 'onboarding.html'
+    });
+  }
+});
+```
+
+Retrieve the user's currently focused tab. Requires Manifest V3. Cannot be used in a content
+script.
+
+```js
+async function getCurrentTab() {
+  let queryOptions = { active: true, currentWindow: true };
+  let [tab] = await chrome.tabs.query(queryOptions);
+  return tab;
+}
+```
+
+Content script asking a background script to navigate the current tab.
+
+```js
+// Content script
+function requestNavigation(url) {
+  chrome.runtime.sendMessage({ action: 'navigate', url });
+}
+
+// Background script
+chrome.runtime.onMessage.addListener(messageListener);
+
+function messageListener(request, sender, _sendResponse) {
+  if (request.action === 'navigate') {
+    // Sanitize URL in to guard against compromised content scripts
+    let url = new URL(request.url);
+    chrome.tabs.update(sender.tab.id, { url: url.toString() });
+  }
+}
+```
+
+### More samples
+
 ![Two tabs in a window](tabs.png)
```

Comment:
I don't see the image on the preview...

Reply:
That image wasn't migrated in the transition to the new site. For the moment I'm just going to delete the image reference.